### PR TITLE
micmac orientation, naming homogeneization, --no-proxy option

### PR DESCRIPTION
**Pull request by mbredif**
This PR adds : 
- Micmac orientation support via the import-ori option
- Micmac orientation test files and test cases in test.sh and test2.sh
- a --no-proxy option, similar to wget, that disables any proxy settings
- homogeneize `name`s across import modes
- refactors code such that importing micmac orientation may call the import of the internal or external intrinsic calibration, via the import-autocal code

### CODE REVIEW

File: micmac_li3ds/import_autocal.py [hunk 1]
```diff
@@ -62,158 +59,174 @@ def get_parser(self, prog_name):
             '--indent', type=int,
             help='number of spaces for pretty print indenting')
         parser.add_argument(
-            'filename',
+            'filename', nargs='+',
             help='the autocal file')
```
Comment (elemoine) [inline micmac_li3ds/import_autocal.py]: What about mentioning in the help message that multiple files may be specified on the command line?

File: micmac_li3ds/import_orimatis.py [hunk 2]
```diff
@@ -68,29 +74,35 @@ def get_parser(self, prog_name):
             '--indent', type=int,
             help='number of spaces for pretty print indenting')
         parser.add_argument(
-            'filename',
+            'filename', nargs='+',
             help='the orimatis file')
```
Comment (elemoine) [inline micmac_li3ds/import_orimatis.py]: You could mention that multiple orimatis files may be specificed on the command line.

File: micmac_li3ds/import_blinis.py [hunk 3]
```diff
@@ -63,91 +60,95 @@ def get_parser(self, prog_name):
             '--indent', type=int,
             help='number of spaces for pretty print indenting')
         parser.add_argument(
-            'filename',
+            'filename', nargs='+',
             help='the blinis file')
```
Comment (elemoine) [inline micmac_li3ds/import_blinis.py]: Mention that multiple blinis files may be specificied?

**Comment by mbredif** [comment]
Here is the PR split into multiple commits, with a final commit to travis-check the 2 test scripts.
I will merge it in a few days if I get no objections.

File: setup.py [hunk 4]
```diff
@@ -10,6 +10,7 @@
     'cliff==2.4.0',
     'requests==2.13.0',
     'pytz==2017.2',
+    'pbr>=2.0.0',
```
Comment (elemoine) [inline setup.py]: Why `pbr`? Is this related to testing on Travis?

**Comment by mbredif** [comment]
I do not know what it is, but the build fails on travis, requiring pbr>=2.0.0...

File: setup.py (hunk 4 above)
Comment (elemoine) [inline setup.py]: `pbr` is a dependency of `cliff`, which is one of our dependencies. But `pbr` shouldn't be a direct dependency for us.

**Comment by elemoine** [comment]
> I do not know what it is, but the build fails on travis, requiring pbr>=2.0.0...

We don't want to depend on `pbr`. It's an indirect dependency for us. Please merge, and I'll tackle this separately.

**Comment by elemoine** [comment]
@mbredif see https://github.com/LI3DS/micmac_li3ds/commit/7e93ab80dd084c1b4dbf4ec8f87d30cc40c2c752. Feel free to cherry-pick it into your branch. Or you can merge your branch as-it and I'll create a separate PR for the `pbr` issue.

**Comment by mbredif** [comment]
Thanks @elemoine for the review and the fix. please file a PR for your dependency fix.
